### PR TITLE
docs: target_match and source_match are DEPRECATED

### PR DESCRIPTION
**Pull request by 7olstoy**
https://prometheus.io/docs/alerting/latest/configuration/#inhibit_rule

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,16 +149,14 @@ route:
...
+  target_matchers: [ severity="warning" ]
   # Apply inhibition if the alertname is the same.
-  # CAUTION: 
-  #   If all label names listed in `equal` are missing 
+  # CAUTION:
+  #   If all label names listed in `equal` are missing
   #   from both the source and target alerts,
   #   the inhibition rule will apply!
-  equal: ['alertname']
+  equal: [ alertname, cluster, service ]
```
Comment (roidelapluie) [inline README.md:159]: Can we keep only alertname in this documentation? Thanks!

Comment (7olstoy) [reply README.md]: As you wish

**Review by roidelapluie — APPROVED** [review]

**Comment by roidelapluie** [comment]
Thanks for your PR. For legal reasons, we require that all commits are signed with a [DCO](https://developercertificate.org/) before we can merge them. See [this blog post](https://www.cncf.io/blog/2016/11/08/cloud-native-software-can-trust/) for considerations around this.

This means that the last line of your commit message should read like:

`Signed-Off-By: Your Name <your@email.address>`

If you are using GitHub through the web interface, it's quickest to close this PR and open a new one with the appropriate line.

If you are using Git on the command line, it is probably quickest to amend and force push. You can do that with
````
git commit --amend --signoff
git push -f $remote $remote_branch_for_pr
````

As always, be careful when force-pushing.

**Comment by 7olstoy** [comment]
Open #2665 due to wrong force push

**Pull request closed without merging**